src/Pie.js

```diff
@@ -46,7 +46,7 @@ const RingShape = props =>
         <ArcShape {...props} startAngle={180} arcAngle={180} />
       </Group>
 
-const Pie = ({ series, colors, radius, innerRadius, backgroundColor,strokeCap}) => {
+const Pie = ({ sections, radius, innerRadius, backgroundColor,strokeCap}) => {
```

Comment:
also we could use `const Pie = ({ sections, radius, innerRadius, backgroundColor,strokeCap, ...rest}) => {` to spread testId and other props to the container

Reply:
I was thinking to make a separate PR for testID support if that's alright